src/ClusterBootstrap/params.py

```diff
@@ -655,6 +656,7 @@
         "Standard_NC12s_v2": {"gpu-type": "P100", "gpu-count": 2},
         "Standard_NC24rs_v3": {"gpu-type": "V100", "gpu-count": 4},
         "Standard_NC24rs_v2": {"gpu-type": "P100", "gpu-count": 4},
+        "default": {"gpu-type": "CPU", "gpu-count": 0},
```

Comment:
Let's call it "None"?, it seems a little bit strange if gpu-type is CPU

Reply:
sure